### PR TITLE
Add school norm

### PR DESCRIPTION
**Pull request by dsquirek**

### CODE REVIEW

File: SchoolNormalization.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+                "confidence": 2
+            }
+        ]
+    }
+}
+```
+
+#Versioning
+-----------
+The response from the School Normalization Service is versioned with the current version being 1.0. The data set used to preform the normalization is also versioned with the current version being 1.0.0, however customers will not have the ability to use older versions of the data set as the updates are done to improve the accuracy of the results rather, and will not impact the response.
```
Comment (matthuhman) [inline SchoolNormalization.md]: perform*

File: SchoolNormalization.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+
+
+#Request Information
+
+
+HTTP method: GET or POST
+Parameters (query/form):
+-        query (required) : The educational instutution information to be queried.
+-        country (optional) : 2 digit ISO country code of eductaionl institution. Accepted country codes are [here](#supported-countries)
```
Comment (matthuhman) [inline SchoolNormalization.md]: educational*

**Comment by matthuhman** [comment]
Looks good once typos are fixed

File: SchoolNormalization.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+- [Supported Countries](#supported-countries)
+
+
+
+#Request Information
+
+
+HTTP method: GET or POST
+Parameters (query/form):
+-        query (required) : The educational instutution information to be queried.
```
Comment (thomasmahoney) [inline SchoolNormalization.md]: institution*

File: SchoolNormalization.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+
+```
+{
+    "data": {
+        "normalized_schools": [
+            {
+                "normalized_school_name": "Georgia Institute of Technology",
+                "id": "53bff579e4b04710d09fa98d",
+                "country": "US",
+                "surface_form": "georgia tech",
```
Comment (thomasmahoney) [inline SchoolNormalization.md]: I just noticed this -- do we want to be exposing surface_form in our service? it's not in P&N, and afaik that's usually something that the data scientists use internally. you can mark it transient if you don't want it to serialize.

File: SchoolNormalization.md (hunk 1 above)
Comment (thomasmahoney) [inline SchoolNormalization.md]: I would reword the second sentence here, I think it's a little confusing. Maybe something like...

"The data set used to perform the normalization is also versioned (currently 1.0.0). This data set is upgraded at will in order to improve accuracy of the classifier. Requests made to this service will always use the latest version of the data set."

File: SchoolNormalization.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+
+```
+{
+    "data": {
+        "normalized_schools": [
+            {
+                "normalized_school_name": "Georgia Institute of Technology",
+                "id": "53bff579e4b04710d09fa98d",
+                "country": "US",
+                "confidence": 2
```
Comment (thomasmahoney) [inline SchoolNormalization.md]: is confidence always an integer? what is the possible range? these are things that IMO aren't clear from the documentation. I think a small section explaining the fields would be good, and/or a different example response that shows multiple schools with varying confidence. if we guarantee results in decreasing order of confidence, we should say that too.

Comment (thefroatgt) [inline SchoolNormalization.md]: If we aren't ordering, we should be, and it should be indicated in the documentation.

Comment (dsquirek) [inline SchoolNormalization.md]: Adam we are ordering by confidence which is a double

**Comment by thomasmahoney** [comment]
:+1:
